### PR TITLE
change border-radius to ha-card-border-radius

### PR DESCRIPTION
**Pull request by tivoo**
Change to reflect border-radius of 20px across the interface for uniformity.

### CODE REVIEW

File: custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-mobile-tapbar/minimalist-mobile-tapbar.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 minimalist-mobile-tapbar:
   # Journal
   state-icon-color: "rgb(var(--color-theme))"
-  border-radius: "20px"
```
Comment (tivoo) [inline custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-mobile-tapbar/minimalist-mobile-tapbar.yaml:5]: Actually need to leave this in for all Minimalist UI custom cards.

File: custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-desktop/minimalist-desktop.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 minimalist-desktop:
   # Journal
   state-icon-color: "rgb(var(--color-theme))"
-  border-radius: "20px"
```
Comment (tivoo) [inline custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-desktop/minimalist-desktop.yaml:5]: Need to leave line 5 in for Minimalist UI custom cards

File: custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-mobile/minimalist-mobile.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 minimalist-mobile:
   # Journal
   state-icon-color: "rgb(var(--color-theme))"
-  border-radius: "20px"
```
Comment (tivoo) [inline custom_components/ui_lovelace_minimalist/lovelace/themefiles/minimalist-mobile/minimalist-mobile.yaml:5]: Need to leave line 5 in for Minimalist UI custom cards

**Review by CM000n — APPROVED** [review]

**Comment by tivoo** [comment]
In the release line 5 is not re-added. This line with the original border-radius should be re-added for custom-cards templates

**Pull request merged**